assets/src/util/dateTime.ts

```diff
@@ -23,6 +23,15 @@ export const formattedTime = (date: Date): string => {
   return formattedHoursMinutes(date.getHours(), date.getMinutes())
 }
 
+export const formattedDate = (date: Date): string => {
+  const displayMonth = date.getMonth() + 1
+  const day = date.getDate()
+  const displayDay = day < 10 ? `0${day}` : `${day}`
+  const displayYear = date.getFullYear()
+
+  return `${displayMonth}/${displayDay}/${displayYear}`
+}
```

Comment:
suggestion (non-blocking): I'm a fan of the [date-fns](https://date-fns.org/docs/Getting-Started) library which has some nice date formatting helpers